jf_agent/main.py

```diff
@@ -107,6 +107,9 @@ def main():
     parser.add_argument(
         '-u', '--until', nargs='?', default=None, help='DEPRECATED -- has no effect'
     )
+    parser.add_argument(
+        '-dr', '--debug-request', type=bool, default=False, help='turn on verbose http logging on errors'
+    )
```

Comment:
nit: should use `store_true` here so we can just provide the flag rather than having to do `--debug-request True`

```suggestion
    parser.add_argument(
        '-dr',
        '--debug-request',
        action='store_true',
        help='turn on verbose http logging on errors',
    )
```